packages/core/src/WebviewDatadog/__utils__/formatAllowedHosts.ts

```diff
@@ -0,0 +1,13 @@
+export function formatAllowedHosts(allowedHosts?: string[]): string {
+    try {
+        if (typeof allowedHosts === 'number') {
+            throw new Error('Input is a number');
+        }
+        if (typeof allowedHosts === 'undefined') {
+            throw new Error('Input is undefined');
+        }
```

Comment:
I believe we could remove these checks if they are not needed for the webview to work, otherwise that's more code to maintain